the-experts-fe/src/App.js

```diff
@@ -1,37 +1,44 @@
-import React, { Component } from "react";
+import React, { useState, useEffect } from "react";
 import logo from "./logo.svg";
 import "./App.css";
+import Select from "./components/select/select";
+import Contractors from "./components/contractors/contractors";
+import Contractor from "./components/contractor/contractor";
 
-class App extends Component {
-  constructor(props) {
-    super(props);
-    this.state = { apiResponse: "" };
-  }
+export default function App() {
+  const [jobsArray, setJobArray] = useState([
+    { id: 1, value: "wood-worker" },
+    { id: 2, value: "plumber" },
+    { id: 3, value: "electric" },
+    { id: 4, value: "painter" }
+  ]);
+  const [jobValue, setJobValue] = useState("");
+  const [selectedContractors, setSelectedContractors] = useState([]);
 
-  callAPI() {
-    console.log(process.env.REACT_APP_testAPI)
-    fetch(process.env.REACT_APP_testAPI)
+  useEffect(() => {
+    fetch(`http://localhost:9000/contractor-results/${jobValue}`)
       .then(res => res.text())
```

Comment:
You should use `res.json()` rather than `res.text()` here, as the content-type of the data you are getting back from the fetch is JSON.